### ISSUE TITLE
Add tests for knit_print output

### ISSUE THREAD

**Issue by mitchelloharawild** [mitchelloharawild/vitae#110]
#108 could have been prevented if `knit_print` tests captured this breaking change.

Adding test examples checking for `knit_print` output should provide reasonably detailed tests for `brief_entries()`, `detailed_entries()` and `bibliography_entries()` outputs.

**Comment by johnaeanderson** [comment]
#108 still isn't fixed for me after grabbing the latest version of vitae.  I'm still getting

Error: No common type for `year` <character> and `year` <double> ... I never had these issues with the previous versions.  I expect that tibble has suddenly changed to only allow one class.

**Comment by mitchelloharawild** [comment]
Yes, this would also be due to changes in tibble. Could you post another issue containing a minimally reproducible example (https://www.tidyverse.org/help/) of this issue?
